src/Http/Headers/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+static readonly Microsoft.Net.Http.Headers.HeaderNames.Protocol -> string!
```

Comment:
Is this the right place for this? Seems like there are many other public `HeaderNames` but they aren't listed here. This is just the place that VS recommended.